docs/development/Running_Acceptance_Tests.md

```diff
@@ -1,29 +1,30 @@
-## Passport Verify Stub Relying Party - Acceptance Tests
+## Passport Verify Stub Relying Party - Acceptance tests
 
 Running the end to end acceptance tests requires:
-* the node app
+
+* the Node.js application
 * a database
-* a verify service provider (VSP)
-* a matching service adapater (MSA)
-* the compliance tool
+* a [Verify Service Provider (VSP)](https://github.com/alphagov/verify-service-provider)
+* a [Matching Service Adapater (MSA)](http://alphagov.github.io/rp-onboarding-tech-docs/pages/msa/msa.html)
```

Comment:
Can we not link to the repo for the MSA here too? Or do we think the docs supercede it as they reference the repo?